### DISCUSSION TITLE
Bug with latest version 1.3.0 of Portal

### DISCUSSION THREAD

**Discussion by theramirobeltran** [chakra-ui/chakra-ui#5066] [General]
version 1.3.0 of portal causing issues

```
/root/project/node_modules/@chakra-ui/portal/dist/chakra-ui-portal.cjs.dev.js:106
        const host = appendToParentPortal ? parentPortal ?? doc.body : doc.body;

SyntaxError: Unexpected token '?'

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1350:14)
      at Object.<anonymous> (node_modules/@chakra-ui/portal/dist/chakra-ui-portal.cjs.js:6:20)
```

**Comment by jojonicho** [comment]
Hi there, I stumbled upon this issue while working with next and chakra-ui, but managed to fix it by using node version 14, previously 12.

```
nvm use 14
```

**Reply by TsengMJ** [reply]
THX! It works for me ~~

**Reply by youminkim** [reply]
Anybody this issue even with Node 14.x?

I have

**Reply by louiejakebell** [reply]
yes i am @youminkim but i am seeing this error using @chakra-ui/icons

**Comment by youminkim** [comment]
I installed chakra-ui/react today. It works fine from the recent Chrome or Safari. However, my iOS 12 safari returns similar error. To reproduce this issue, check this url with iOS 12 safari browser https://chakra-ui.com/docs/overlay/modal

After I re-installed chakra old version it works fine in the old iOS safari. 

```
npm i @chakra-ui/react@1.6.12
```

![IMG_0052](https://user-images.githubusercontent.com/317132/141484204-2404bff0-2435-4ea2-ba8d-56ba3e9ca0ae.PNG)
